### PR TITLE
Addresses BACKLOG-37996: WCAG main page zoom

### PR DESCRIPTION
**Pull request by mbrasil**
@pentaho/wcag and @pentaho/millenniumfalcon please review

Main PR: https://github.com/pentaho/pentaho-platform-plugin-interactive-reporting/pull/1015

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 12m 30s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **436**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-998/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 1]
```diff
@@ -7222,196 +7250,312 @@ span.gwt-RadioButton label {
...
+  background-position: initial;
 }
 
 .tundra .dijitEditorIconIndent {
+  --icon-height: 22;
+  --icon-width: 22;
+
+  height: calc(1px * var(--icon-height));
+  width: calc(1px * var(--icon-width));
+
```
Comment (davidmsantos90) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:7302]: similar to what was done with other icon styles, at least the `vars` and `height/width` could be aggregated in one selector. eg: `.tundra :is(.dijitEditorIcon.dijitEditorIconUndo, ..., ...) {`

also take a look at the following icon styles (bellow), there is a lot of repetition that can be avoided if we aggregate them 💪🏼

**Review by davidmsantos90 — CHANGES_REQUESTED** [review]

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 2]
```diff
@@ -4758,11 +4782,11 @@ div.dijitDateTextBox.dijitComboBox {
 }
 
 .sidePanelIconOpen {
-  background: url('images/16x16_right.png') no-repeat;
+  background-image: url('images/16x16_right.png');
 }
 
 .sidePanelIconClose {
-  background: url('images/16x16_left.png') no-repeat;
+  background-image: url('images/16x16_left.png');
```
[the remark this answers is not in the document]
Comment (mbrasil) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:4789]: You're right, but I had this covered. There is a class called `sidePanelIconCommon` that has all things common between `sidePanelIconOpen` and `sidePanelIconClose` and it's in there.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 12m 38s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **436**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-998/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 12m 24s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **436**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-998/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by davidmsantos90 — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 12m 6s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **436**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-998/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Pull request merged**